src/common/components/help/help-install-snap.js

```diff
@@ -41,6 +41,11 @@ export default class HelpInstallSnap extends Component {
           <CopyToClipboard
             copyme={ command }
           />
+          <Tweet
+            text={ `Install ${name} in seconds on Linux OSes:\n`
```

Comment:
Per review, pass the text to the component, which makes more sense in terms of editing content in the parent etc, as well as making it more flexible.